### PR TITLE
Added a servlet that handles the images uploaded by users

### PR DESCRIPTION
**Pull request by SBenjaminVR**
Now you can upload an image in your posts

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/MessageServlet.java [hunk 1]
```diff
@@ -76,15 +78,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String userEnteredContent = request.getParameter("text");
-    
+    String test = request.getParameter("text");
```
Comment (raujimenez) [inline src/main/java/com/google/codeu/servlets/MessageServlet.java]: I think you can clean this code up a little bit by removing the test variable. You might have been playing around with the code and left it here accidentally. It works well, just send a new commit that fixes this very slight issue. Other than that it looks nice and it compiles!

**Review by raujimenez — COMMENTED** [review]
Nice work! I left a comment in the MessageServlet.java file. Its a very slight change but it will help clean up the code a little bit. other than that good job. Here is a picture of it working.
image

File: src/main/java/com/google/codeu/servlets/MessageServlet.java (hunk 1 above)
Comment (SBenjaminVR) [reply src/main/java/com/google/codeu/servlets/MessageServlet.java]: Yes, I forgot about that, It is fixed now. Thanks

**Review by lsj1124 — APPROVED** [review]
Cool! It looks good to me!

**Pull request merged**